src/main/java/core/basesyntax/service/ParserImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import java.util.Arrays;
+
+public class ParserImpl implements Parser {
+    @Override
+    public Transaction parseLine(String line) {
+        String[] splittedLine = line.split(",");
+        isValid(splittedLine);
+        return new Transaction(splittedLine[0],
+                new Fruit(splittedLine[1]),
+                Integer.parseInt(splittedLine[2]));
+    }
+
+    private void isValid(String[] value) {
```

Comment:
```suggestion
    private void validate(String[] value) {
```
`isValid` is supposed to return boolean